build.rs

```diff
@@ -0,0 +1,97 @@
+extern crate prost_build;
+use reqwest::{header::USER_AGENT, Url};
+use std::{
+    env,
+    io::{Cursor, Result},
+};
+
+const DCL_PROTOCOL_REPO_URL: &str =
+    "https://api.github.com/repos/decentraland/protocol/contents/proto/decentraland";
+const FRIENDSHIP_PROTO_PATH: &str = "/social/friendships/friendships.proto";
+// Modify this value to update the proto version, it is the commit sha from protocol repo used for downloading the proto file
+const FRIENDSHIPS_PROTOCOL_VERSION: &str = "c858f8a15323fd86a18474c8b650deea28507d3b";
+const DEFINITIONS_FOLDER: &str = "ext-proto";
+const PROTO_FILE_DEST: &str = "ext-proto/friendships.proto";
+
+fn main() -> Result<()> {
+    download_proto_from_github()?;
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    println!("cargo:rerun-if-changed=ext-proto/friendships.proto");
+
+    let mut prost_config = prost_build::Config::new();
+    prost_config.protoc_arg("--experimental_allow_proto3_optional");
```

Comment:
I'm not sure if this is needed here, I mean on CI/prodcution. I think @agusaldasoro you need it on your machine due to your `protoc` version/binary. At https://github.com/decentraland/quests there are `optional` fields and we've not needed that flag to make it run.

Reply:
In my local works, but it doesn't work on the Docker build, maybe I should change the protoc version that Docker is installing?